test/fixtures/users.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
delete this file
you dont need do test

Reply:
![Screenshot from 2021-08-30 20-21-50](https://user-images.githubusercontent.com/87750131/131338120-1d3609ad-d507-4c7b-800a-6c95ff7d1574.png)